docs/narr/startup.rst

```diff
@@ -63,17 +68,20 @@ Here's a high-level time-ordered overview of what happens when you press
    will have a single ``[app:main]`` section in it, and this will be the
    application served.
 
+.. index::
+   pair: logging; startup
+
+.. _startup_logging_initialization:
+
 #. The framework finds all :mod:`logging` related configuration in the ``.ini``
    file and uses it to configure the Python standard library logging system for
-   this application.  See :ref:`logging_config` for more information.
+   the application.  See :ref:`logging_config` for more information.
 
-#. The application's *constructor* named by the entry point referenced on the
-   ``use=`` line of the section representing your :app:`Pyramid` application is
-   passed the key/value parameters mentioned within the section in which it's
-   defined.  The constructor is meant to return a :term:`router` instance,
-   which is a :term:`WSGI` application.
+#. The application's entry point :ref:`found above <entry_point_discovery>`, usually the entry point referenced on the above mentioned ``use=`` line, is the application's *constructor*.
```

Comment:
Remove redundancy. Fix grammar.

```
The application's entry point, usually the entry point referenced on the above mentioned ``use=`` line, is the application's *constructor*.
```

Reply:
We don't define a term for "constructor" in the Glossary. It would be good to add one.